<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My_first_PyQt_GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="Cam_image">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Camera</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="5">
        <widget class="QLineEdit" name="CamID_Edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="Camhandle_Edit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Camera ID :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="InitCam_Button">
         <property name="text">
          <string>Init</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Handle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="CloseCam_Button">
         <property name="text">
          <string>Release</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="StartCam_Button">
         <property name="text">
          <string>Start acquisition</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="StopCam_Button">
         <property name="text">
          <string>Stop acquisition</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Pump</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Pump_Button">
         <property name="text">
          <string>Start injection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Volume_Edit">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Flow-rate :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FlowRate_Edit">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="CloseGUI_Button">
      <property name="text">
       <string>Close GUI</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>DAQ connection</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="DAQinit_Button">
         <property name="text">
          <string>Init DAQ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DAQstop_Button">
         <property name="text">
          <string>Release DAQ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_open">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile_open"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
